sydneywait/AppalachianHarvest
# Edit and Delete Producers

# Description
Users can now edit and delete producers


## Related Ticket(s)
https://github.com/sydneywait/AppalachianHarvest/issues/7



## Steps to Test
--Pull down the branch sw-edit-delete-producers
--navigate to  'producers'
--practice adding, editing and deleting producers
--check all other functionality with products to make sure it is still working

# Checklist:
